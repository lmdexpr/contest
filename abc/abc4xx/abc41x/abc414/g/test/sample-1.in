6 3
0 20 50 90 110 150
1 2 5 6 100
1 1 2 3 10000
6 6 1 2 30
