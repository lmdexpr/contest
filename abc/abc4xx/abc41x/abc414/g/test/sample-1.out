410 10050 -1 210 250
